ledgerwallet/ledgerctl.py

```diff
@@ -162,9 +171,15 @@ def list_apps(get_client, remote, url, key):
     help="Delete using application hash instead of application name",
     is_flag=True,
 )
+@click.option(
+    "-d",
+    "--dump",
```

Comment:
Could you rename en offline to be iso with ledgerblue parameter naming?

Reply:
Yes

Reply:
Done.